java/ql/test/experimental/query-tests/security/CWE-927/SensitiveBroadcast.java

```diff
@@ -0,0 +1,130 @@
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+class SensitiveBroadcast {
+
+    //Tests broadcast of access token with intent extra.
+    public void sendBroadcast1(Context context, String token, String refreshToken) {
```

Comment:
Use `BAD` and `GOOD` or `OK` to indicate which cases are a problem that should be detected and which should not.

Reply:
Added <code>BAD</code> and <code>GOOD</code> to the comments of all test cases